src/EventTickets/resources/admin/components/EventTicketsListTable/EventTicketsListTable.module.scss

```diff
@@ -1,3 +1,26 @@
+.listTable {
+    :global {
+        table {
+            tbody {
+                tr {
+                    td {
+                        &:nth-child(4) {
+                            p {
```

Comment:
Could we simplify how these styles get applied?

Reply:
I have now formatted it like other tables by applying a custom class to the cell. See.42fb704507bd885eade56327faf6aa9c54bea38b